core/common/src/main/java/alluxio/RuntimeConstants.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
it was in `Constants.java` before, but causing the problem of introducing circular dependency